install/install.sh

```diff
@@ -1,40 +1,45 @@
 #!/bin/sh
 
-BUILD_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+BUILD_DIR="$( cd "$(dirname "$0")" || exit ; pwd -P )"
 TEMP="${BUILD_DIR}"/temp
 ROOT="${BUILD_DIR}"/..
-VERSION=`cat ${ROOT}/io_scene_nif/VERSION`
+VERSION=$(cat "${ROOT}"/io_scene_nif/VERSION)
 NAME="blender_nif_plugin"
 
-function find_blender() {
+find_blender() {
     for BLENDER_VERSION in 2.79 2.78 2.77 2.76 2.75
```

Comment:
I'm fairly sure we're using API that is not compatible this far back. I say support 2.79 exclusively.

Reply:
Good point as once we jump over to 2.8 we won't be going back anyways.

Reply:
Done